package.json

```diff
@@ -28,7 +28,7 @@
     }
   },
   "peerDependencies": {
-    "webpack": ">=0.9 <2 || ^2.1.0-beta || ^2.2.0"
+    "webpack": ">=0.9 <2 || ^2.1.0-beta || >=2.2.0 <4"
```

Comment:
That would include `2.0.x` again and disallow `0.9`, which I'd consider a breaking change (semver major).